packages/twenty-front/codegen.cjs

```diff
@@ -5,13 +5,33 @@ module.exports = {
     (process.env.REACT_APP_SERVER_BASE_URL ?? 'http://localhost:3000') +
     '/graphql',
   documents: [
+    // Exclude system operations (now handled by metadata codegen)
+    '!./src/modules/auth/**',
+    '!./src/modules/billing/**',
+    '!./src/modules/workspace/**',
+    '!./src/modules/workspace-member/**',
+    '!./src/modules/workspace-invitation/**',
+    '!./src/modules/users/**',
+    '!./src/modules/settings/accounts/**',
+    '!./src/modules/settings/admin-panel/**',
+    '!./src/modules/settings/lab/**',
```

Comment:
Since this is meant to become smaller the negative form/blacklist doesn't make as much sense imo, it should rather be a whitelist